pkgdb2/api/packages.py

```diff
@@ -1287,9 +1291,11 @@ def api_package_request():
             cnt += 1
         collections.append(collection)
 
+    namespaces = pkgdblib.get_status(SESSION, 'namespaces')['namespaces']
```

Comment:
I see now -- this is getting all of the namespaces in order to populate a drop-down menu, correct?  If that is correct, then this plural usage is also correct.  (my bad).